frontend/src/components/LogIn/LogIn.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { AxiosResponse } from 'axios';
+
+import {
+  Button, 
+  CssBaseline,
+  Link,
+  Grid,
+  Typography,
+  Container,
+  FormHelperText,
+  Snackbar
+} from '@material-ui/core';
+import MuiAlert, { AlertProps }  from '@material-ui/lab/Alert';
+
+import StyledTextField from '../StyledTextField'
+import useStyles from './LogIn.style';
+import BaseService from '../../app/baseService';
+
+export interface LogInProps {
+
+};
```

Comment:
seems unnecesary (unless you expect there might be some changes in the future here)

Reply:
For now, I would leave it as it is, because if we want to change something, it will be at hand, and it does not bother whether it is or not.